Stack/Opc.Ua.Core/Stack/Transport/ITransportListener.cs

```diff
@@ -62,6 +62,7 @@ ITransportListenerCallback callback
         /// </summary>
         void CertificateUpdate(
             ICertificateValidator validator,
+            bool sendCertificateChain,
```

Comment:
in order to avoid the interface change, would it be possible to signal the flag by setting serverCertificateChain to null?

Reply:
I could change this for shure if wanted, while still adhering to the current behaviour after certificate update.
 Advantage of my solution is that protected property ServerCertificateChain of the  TcpServerChannel is still set to the chain after a certificate update. 

Please advise.

Reply:
I would prefer a solution without interface change as well, but do have to admit that I do not fully understand the advantage you see. Maybe we can discuss this in our next meeting.

Reply:
I reverted the interface change to adhere to the existing behaviour on certificate update